### ISSUE TITLE
Pretrained models

### ISSUE THREAD

**Issue by hhheeexxxuuu** [vickyFox/Region-wise-Inpainting#9]
Thank you for your work! You have provided two pretrained models: Pretrained model and ijcai Pretrained model.  What is the difference between them? I have trained the model with 28000 face dataset and using wgan-gp loss  with ijcai Pretrained model when the value of stage is one . But the model does not converge as follows. What' more, I think the code of wgan_gp loss should be change a little.  Another question is how many iterations do you used? Could you give me some advice?
![2](https://user-images.githubusercontent.com/39242394/88642452-17042780-d0f3-11ea-97e3-88588a014d9a.png)
![无标题](https://user-images.githubusercontent.com/39242394/88642493-25524380-d0f3-11ea-96cb-174821edbe61.png)

**Comment by shihaobai** [comment]
Thanks for your attention! The ijcai pretrained model is the pretrained model for our previous work, "Coarse-to-Fine Image Inpainting via Region-wise convolutions and Non-local correlation".  You'd better train our model from the beginning.  In the first stage, we pretrain the model for 9 epoches without adversarial loss and then train the model with adversarial loss for 19 epoches. What's more, we didn't normalize the loss function during training. Therefore, the value of loss is large when it finally converges.

**Comment by hhheeexxxuuu** [comment]
Thank you very much! I will try again.
